packages/communication-react/review/communication-react.api.md

```diff
@@ -1270,32 +1270,6 @@ export interface HorizontalGalleryStyles extends BaseCustomStyles {
     previousButton?: IStyle;
 }
 
-// @internal
-export const _IdentifierProvider: (props: _IdentifierProviderProps) => JSX.Element;
-
-// @internal
-export interface _IdentifierProviderProps {
```

Comment:
We actually exported this from our GA package

Reply:
Would recommend splitting this change out of this PR and discussing it internally. I think this was only partially "accidental".

Reply:
Great point, will do

Reply:
updated